transformations/sentence_reordering/transformation.py

```diff
@@ -0,0 +1,61 @@
+import random
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+# for sent tokenizer
+import spacy
+
+nlp = spacy.load("en_core_web_sm")
```

Comment:
Please move this inside the constructor too.

Reply:
example: https://github.com/GEM-benchmark/NL-Augmenter/blob/main/transformations/replace_numerical_values/transformation.py

Reply:
Updated.